### DISCUSSION TITLE
Feedback usability

### DISCUSSION THREAD

**Discussion by improve-code** [i3/i3#6174] [Ideas]
I would like to give a short feedback:

I could not easily get

- screen brightness (via fn button + F5/F6)
- screenshots (via print button)

working.

These features are essential for me.

I made two unsuccessful attempts. Then I decided for the moment not to put any further work in to it.
So I do not use i3, despite liking it.

**Comment by okraits** [comment]
Those things are not among the responsibilities of a window manager. i3 is a window manager, it manages windows and nothing else.
You can easily get those things working with external tools like xbacklight and scrot.

**Comment by improve-code** [comment]
Thank you for your fast answer.

I installed external tools, but did not all get work within a reasonable time.
I.e. screenlock easily worked.


Perhaps on the i3 website could be
- made more clear  what i3 is, and what not (as you stated)
- offered an working solution / easy installation manual for the basic non-window manager functions.
